ecs_fargate/datadog_checks/ecs_fargate/data/conf.yaml.default

```diff
@@ -319,3 +319,9 @@ instances:
     ## Whether or not to persist cookies and use connection pooling for increased performance.
     #
     # persist_connections: false
+
+    ## @param min_collection_interval - number - optional - default: 20
+    ## This changes the collection interval of the check. For more information, see:
+    ## https://docs.datadoghq.com/developers/write_agent_check/#collection-interval
+    #
+    min_collection_interval: 20
```

Comment:
Should this line be commented? All the others are.

Reply:
No because it's not default, that's the change in `spec.yaml`

Reply:
But isn't it now the default after this change? https://github.com/DataDog/integrations-core/pull/10409/files#diff-bb54b8b2e5d4f326e382029c94bf69b6fbc1c4b5d97759fd806f9d9a9fcf8d77L100

Reply:
From what I understood, it's not the general default but an override for this check, hence the non-commented line.